[gameplay]/playerblips/server/main.lua

```diff
@@ -0,0 +1,71 @@
+local useTeams = get("*use_team_colors") == "true" and true or false
+local useNametags = get("*use_nametag_colors") == "true" and true or false
```

Comment:
```suggestion
local useTeams = get("*use_team_colors") == "true"
local useNametags = get("*use_nametag_colors") == "true"
```
Nit: this is the same

Reply:
Fixed